index.html

```diff
@@ -39,7 +39,10 @@
 
     <mgt-login></mgt-login>
     <mgt-person person-query="me" view="twoLines" person-card="hover" show-presence></mgt-person>
+    <br />
+    <mgt-people-picker allow-any-email></mgt-people-picker>
```

Comment:
I don't think we'll need this for our index.html page, using the storybook example should be good enough.